libs/core/serializable_function/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2019 The STE||AR-Group
+#
+# SPDX-License-Identifier: BSL-1.0
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+
+# Default location is $HPX_ROOT/libs/serializable_function/include
+set(serializable_function_headers
+    hpx/serializable_function/serialization/detail/serializable_basic_function.hpp
+    hpx/serializable_function/serialization/detail/vtable/serializable_function_vtable.hpp
+    hpx/serializable_function/serialization/detail/vtable/serializable_vtable.hpp
+    hpx/serializable_function/serialization/serializable_function.hpp
+    hpx/serializable_function/serialization/serializable_unique_function.hpp
+)
+
+# Default location is $HPX_ROOT/libs/serializable_function/include_compatibility
+# cmake-format: off
+set(serializable_function_compat_headers
+    hpx/util/serializable_function.hpp => hpx/serializable_function.hpp
+    hpx/util/serializable_unique_function.hpp => hpx/serializable_function.hpp
```

Comment:
`hpx/serializable_function.hpp` doesn't actually exist, does it? `GLOBAL_HEADER_GEN` generates `hpx/modules/serializable_function.hpp`. So far `hpx/functional.hpp` only included the non-serializable `hpx::function`. Should it include both?

Reply:
I just copied and pasted the compat headers stuff from functional and changed the name accordingly.  I guess we can just get rid of this ass it's not really needed for the new module.